modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro

```diff
@@ -49,6 +49,10 @@ definition {
 		Click(locator1 = "Button#SUBMIT");
 	}
 
+	macro goBack {
```

Comment:
should we specify in the name that is using history or back button?

Reply:
Yes, We can do it. In fact, we can have both methods. goBackUsingHistory and goBackUsingBrowserArrow (when poshi task is done)

Reply:
make sense!